regulations/generator/html_builder.py

```diff
@@ -86,7 +86,14 @@ def process_node(self, node):
             node['label'], node['node_type'])
 
         node['list_level'] = list_level
-        node['list_type'] = list_type
+
+        # exception for situations in which we have unnumbered definitions
+        # unnumbered defs have the last part of their label in CamelCase
+        # and the word "means" in their text
+        if re.search('([A-Z][a-z]+)+', node['label'][-1]) and re.search('means', node['text']):
```

Comment:
Keep in mind that this loop gets called _a lot_. Regexs like this may cause some significant slow down.

It looks like you'll have access to the layer data below -- I wonder if there's a more semantic way to get what you want done. Maybe check if the paragraph marker layer has no entry for this node? That'd imply that the paragraph should have no markers, hey? You'd need to break through some of the interfaces below, but you wouldn't need to check for magic works.